### PR TITLE
Fix unbroken string issue for Primary Hero Shelf

### LINKED ISSUE #8522: Layout issues in the primary hero caused by unbroken strings
Follow up for #8305 

### Describe the problem and steps to reproduce it:
1. Open AMO -dev homepage
2. Refresh the page until you get the primary hero with an unbroken string (see my example below)

### What happened?
Unbroken strings are not wrapped and, because of that, the primary hero section is too wide

### What did you expect to happen?
Unbroken string should be wrapped

**Notes:**
- this is unlikely to happen in production, hence it's a low priority issue

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #8522 

Before:

Screenshot 2019-08-30 10 34 47

After:

Screenshot 2019-08-30 10 34 08

### CODE REVIEW

File: src/amo/components/HeroRecommendation/styles.scss [hunk 1]
```diff
@@ -84,6 +84,7 @@ $overlayShapeZIndex: 1;
   font-size: $font-size-l;
   line-height: 1.188;
   margin: 14px 0 0 0;
+  word-wrap: anywhere;
```
Comment (kumar303) [inline src/amo/components/HeroRecommendation/styles.scss]: Can you switch it to the standard version, which is [`overflow-wrap`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap)? The `anywhere` value landed in Firefox 65 which is pretty recent but that's probably fine.

**Comment by willdurand** [comment]
Related to this issue: https://github.com/mozilla/addons-frontend/issues/8396

**Comment by bobsilverberg** [comment]
> Related to this issue: #8396, especially:
> 
> > overflow-wrap: anywhere is not cross browser looking at developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#Browser_compatibility

Unfortunately it appears that, in Firefox anyway, `anywhere` is the only value that works in this scenario, so I think we're stuck with it.

@kumar303 It looks kind of like you reviewed this, but it's still awaiting an approval from someone.

**Comment by muffinresearch** [comment]
> > Related to this issue: #8396, especially:
> > > overflow-wrap: anywhere is not cross browser looking at developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#Browser_compatibility
> 
> Unfortunately it appears that, in Firefox anyway, `anywhere` is the only value that works in this scenario, so I think we're stuck with it.
> 

@bobsilverberg Did `word-break: break-word;` not work instead?

**Comment by willdurand** [comment]
> @bobsilverberg Did `word-break: break-word;` not work instead?

@bobsilverberg ^ I am interested too :p

**Comment by bobsilverberg** [comment]
> > @bobsilverberg Did `word-break: break-word;` not work instead?
> 
> @bobsilverberg ^ I am interested too :p

I replied to Stuart about this, but I did it via email. I'm not sure why I thought that was best?

Here's what I said:

It turns out that, according to MDN, `word-break: break-word` is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#Values, so I used the suggested `overflow-wrap` instead. Unfortunately, although `overflow-wrap: break-word` exists, it doesn't work in this case, whereas `overflow-wrap: anywhere` does.

This is why we ended up with `overflow-wrap: anywhere` instead of `word-break: break-word`.

**Pull request merged**